examples/collections/choropleth.py

```diff
@@ -41,6 +37,16 @@ def add(P, color):
         polys.append(p, color=color)
 
 
+# Create the canvas.
+canvas = app.Canvas(size=(800, 800), keys='interactive')
+gloo.set_viewport(0, 0, canvas.size[0], canvas.size[1])
+gloo.set_state("translucent", depth_test=False)
+
+panzoom = PanZoomTransform(canvas)
+paths = PathCollection(mode="agg+", color="global", transform=panzoom)
+polys = PolygonCollection("raw", color="local", transform=panzoom)
+paths.update.connect(canvas.update)
```

Comment:
Paths are all black while color are local to each poly.
Note that in glumpy I get different colors for each counties (not state). There may be a bug in parsing the geo file (I used a topojson file in glumpy).